lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+
+attr_reader :list
+
+
+  def initialize
+    @list = { wings: 12,
+              chicken: 9,
+              special: 15}
+  end
+
+  def print_menu
```

Comment:
YAGNI - no presentation layer was asked for here, and this prsumes that a UI would want a string in this format.